### PR TITLE
Citation fixes for 86 cases

### PR DESCRIPTION
**Pull request by quevon24**
These fixes are for incorrect ocr citation that contains non-citation related text, such as disposition, opinions, judge names, case names, etc.

e.g.

- Affirmed
- Affirmed and remanded.
- BAKER, Judge
- Held: Affirmed
- HOFFMAN, Senior Judge
- No error.
- Image here

### CODE REVIEW

File: data/harvard/missing-files.json [hunk 1]
```diff
@@ -1,481 +1,568 @@
...
+        "data/harvard\\law.free.cap.abb-pr.10\\457.1966867.json",
+        "data/harvard\\law.free.cap.abbn-cas.20\\385.1958397.json",
+        "data/harvard\\law.free.cap.ad.242\\737.5834501.json",
+        "data/harvard\\law.free.cap.ad.242\\749.5824702.json",
+        "data/harvard\\law.free.cap.ad.66\\1.4865239.json",
+        "data/harvard\\law.free.cap.ad.66\\12.4865757.json",
+        "data/harvard\\law.free.cap.ad.66\\14.4865567.json",
+        "data/harvard\\law.free.cap.ad.66\\18.4867078.json",
+        "data/harvard\\law.free.cap.ad.66\\21.4864107.json",
+        "data/harvard\\law.free.cap.ad.66\\7.4868211.json",
```
Comment (flooie) [inline data/harvard/missing-files.json:14]: something looks off with this file.

**Review by flooie — CHANGES_REQUESTED** [review]
Not much to add- this is great.  but we need to figure out the missing files file.

File: data/harvard/missing-files.json (hunk 1 above)
Comment (quevon24) [reply data/harvard/missing-files.json:14]: I think it's because i'm running windows 10, let me run the command with WSL

Comment (flooie) [reply data/harvard/missing-files.json:14]: must be - lets include a new command for windows and add it to the docs (assuming that its explained)

**Pull request merged**